piff/basis_interp.py

```diff
@@ -24,6 +24,48 @@
 from .interp import Interp
 from .star import Star
 
+import jax
+# If uncommented, the following line will make the code run in double precision
+# and have identical results as running the original code in double precision.
+# jax.config.update("jax_enable_x64", True)
```

Comment:
I think we always want double precision.  Our matrices are often moderately ill-conditioned, and in my experience, it's rarely worth the headaches of trying to make sure things always converge nicely with single-precision floats.  Even if it works for some set of use cases, one the breaks is usually right around the corner.  So I think we need this line to be uncommented.